src/main/resources/api/openapi.yaml

```diff
@@ -2817,6 +2817,33 @@ paths:
               schema:
                 $ref: '#/components/schemas/ApiError'
 
+  /transactions/bytes:
+      post:
+        summary: Submit an Ergo transaction given as hex-encoded transaction bytes to unconfirmed pool to send it over the network
+        operationId: sendTransactionBytes
```

Comment:
I would call it `sendTransactionAsBytes` as this will be visible in swagger and it is self-explanatory

Reply:
Makes sense, renamed